locales/en/translation.json

```diff
@@ -460,6 +460,7 @@
   "Remember me": "",
   "Remove": "",
   "Remove People": "",
+  "rescuecarbs": "Rescuecarbs",
```

Comment:
Warn: Tideline use "rescuecarbs" fot the translation, viz: "Rescuecarbs".

Reply:
let's stick to 'rescuecarbs'